### PR TITLE
Some dev environment QoL improvements

### PR DESCRIPTION
**Pull request by eseliger**
The three commits address three things I found could improve the experience of running `dev/start`.

### CODE REVIEW

File: enterprise/dev/Procfile [hunk 1]
```diff
@@ -10,7 +10,7 @@ nginx: nginx -p . -g 'daemon off;' -c $PWD/dev/nginx.conf 2>&1 | grep -v 'could
 caddy: ./dev/caddy.sh run --watch --config=dev/Caddyfile
 # TODO gulp doesn't respect C-c https://github.com/sourcegraph/sourcegraph/issues/13291
 # Fix this and you get to remove the badge of unix shame! :)
-web: ./dev/badge-of-unix-shame.sh ./node_modules/.bin/gulp --color dev
+web: ./dev/badge-of-unix-shame.sh ./node_modules/.bin/gulp --silent --color dev
```
Comment (felixfbecker) [inline enterprise/dev/Procfile:13]: Do we still get some indication with this when things are done?

**Review by felixfbecker — APPROVED** [review]

File: enterprise/dev/Procfile (hunk 1 above)
Comment (eseliger) [reply enterprise/dev/Procfile:13]: Webpack still prints its status notifications, do you think that would be enough?

**Review by mrnugget — APPROVED** [review]
Very nice!

**Review by chrispine — APPROVED** [review]
LGTM

**Pull request merged**